pod/perldelta.pod

```diff
@@ -27,6 +27,14 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 Defined-or and truth-or default expressions to signatures
```

Comment:
perlop calls the operator `logical-or` not `truth-or`, but also I feel the word assignment ought to be in there to differentiate the `defined-or //` operator from the `defined-or assignment operator //=`

Reply:
A good idea. Reworded and force-pushed